src/Bladeburner/Bladeburner.ts

```diff
@@ -1201,11 +1201,9 @@ export class Bladeburner {
             if (this.logging.general) {
               this.log(`${person.whoAmI()}: Incited violence in the synthoid communities.`);
             }
-            for (const cityName of Object.values(CityName)) {
-              const city = this.cities[cityName];
-              city.chaos += 10;
-              city.chaos += city.chaos / (Math.log(city.chaos) / Math.log(10));
-            }
+            const city = this.getCurrentCity();
+            city.changeChaosByCount(10);
+            city.changeChaosByCount(city.chaos / Math.log(city.chaos) / Math.log(10));
```

Comment:
This part is the logical change. The scaling stays the same in this PR, but the penalty now only affects the player's current city.

Also, the change to `changeChaosByCount` handles integer safety with a clamp that defaults to `Number.MAX_VALUE`.

Reply:
To continue on the discord discussion: I vote for combining the two logics. This is not a perfect, final solution, but merely something that I'm confident will move this PR forward and allow us to make adjustments that have a positive effect on playability.

- give the old, heavy penalty to the current city, but not others
- apply the flat penalty to all cities

This way the cost of generating contracts remains global, but the escalating effect only stays on the one you are currently looking at (ie. you won't accidentally end up with your non-current city having 1e50 chaos in two days).

Reply:
I request separating this commit out to a new PR and reverting it in this one so we can merge the surrounding refactors.